lib/plugins/aws/package/compile/events/stream/index.test.js

```diff
@@ -37,6 +37,81 @@ describe('AwsCompileStreamEvents', () => {
   });
 
   describe('#compileStreamEvents()', () => {
+    it('should throw an error if the "consumer" property is not a string or a boolean', () => {
```

Comment:
It's fine if such errors surface with AWS CF validation, we don't need to double validate them here (doing so, we may introduce other issues, and prevent well configured setup from working)